config/environments/test.rb

```diff
@@ -43,4 +43,6 @@
   # config.assets.compile = true
   # config.assets.js_compressor = :uglifier
   config.active_job.queue_adapter = :sidekiq
+  # Configure the drafts strorage directory
+  config.drafts_storage_dir = '/tmp/drafts/'
```

Comment:
Let's just default to putting it in the Rails tmp directory, which will be more convenient. I had the system tmp on the brain because of Travis.